application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -68,16 +68,16 @@ public static String escapeMarkdown(String text) {
      * @return Formatted string for the mentioned slash command
      * @throws IllegalArgumentException when the command isn't found in the guild
      */
-    public static RestAction<String> mentionSlashCommand(Guild guild, String commandPath) {
-        String commandName = commandPath.split(" ", 2)[0];
+    public static RestAction<String> mentionSlashCommand(Guild guild, String... commandPath) {
+        String commandName = commandPath[0];
```

Comment:
now it can crash if not given any value though. u gotta make the first one mandatory by `String commandName, String... subCommands` or similar

Reply:
Right, sorry I forgot about that